controllers/storagecluster/initialization_reconciler_test.go

```diff
@@ -99,7 +100,15 @@ func createUpdateRuntimeObjects(cp *Platform) []runtime.Object {
 		}
 		updateRTObjects = append(updateRTObjects, cos)
 	}
-
+	// Create 'route' only for non-cloud platforms
+	if !avoidObjectStore(cp.platform) {
+		cos := &routev1.Route{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "ocsinit-cephobjectstore",
```

Comment:
Can we do without the `init`?
```suggestion
				Name: "ocs-cephobjectstore",
```

Reply:
The name of the cephobject store in the cephobjectstore_test.go is ocsinit-cephobjectstore. The test sets the route name to the same name as the cephobjectstore for the tests.

On an actual cluster, the route name will be ocs-storagecluster-cephobjectstore.